### DISCUSSION TITLE
Open Space at PyCon

### DISCUSSION THREAD

**Discussion by samuelcolvin** [pydantic/pydantic#4008] [General]
Sorry for the late notice.

I'm going to be at PyCon all week and I'm thinking of hosting an "open space". Please let me know here if you're interested in attending.

Depending on how many people are interested, I can give a short presentation about the plans for pydantic v2.

Regardless of whether I manage to organise something; if you're interesting in pydantic, feel free to come and chat to me. Message me on twitter to organise a meeting.

**Comment by adriangb** [comment]
I am interested and will be there!

**Reply by dltacube** [reply]
Same here!

**Comment by samuelcolvin** [comment]
Open Space for #pydantic tomorrow (Saturday 30th) at 3pm - #PyConUS2022 Room 250AB.

"Pydantic - What's coming in V2 & a review of building python extensions with Rust"

#PyConOpenSpace #pycon #OpenSource #Python
